_layouts/default.html

```diff
@@ -12,30 +10,35 @@
     <title>{{ page.title }}</title>
 
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
-    <link href="{{ "/jumbotron.css" | absolute_url }}" rel="stylesheet">
+    <link href="jumbotron.css" rel="stylesheet">
   </head>
 
   <body>
 
     <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
       <a class="navbar-brand" href="/">Brighton ALT NET</a>
-      <a class="nav-link" href="/directory.html">Directory</a>
-      <a class="nav-link" href="/meetings.html">Meetings</a>
+      <div class="collapse navbar-collapse">
+        <ul class="navbar-nav">
+          <li class="nav-item"> <a class="nav-link" href="/directory.html">Directory</a></li>
+          <li class="nav-item"> <a class="nav-link" href="/meetings.html">Meetings</a></li>
+        </ul>
+      </div>
     </nav>
 
     <main role="main">
-      <div class="container">
-        <br/>    
-      </div>
 
-      <div class="container">
-          <h1>{{ page.title }}</h1>
```

Comment:
Now there's no page title at the top of each page, it's a little hard to know where one is on the site. I can see why this looks crappy from a design point of view, but could we put it back somewhere?

Reply:
Oops, I didn't realise I removed that. I'll put it back

Reply:
No worries :)